### ISSUE TITLE
Millie Weaver - Fanart Broken...Also suggest we add other personalities? Perhaps Savanna Hernandez and others?

### ISSUE THREAD

**Issue by ilikenwf** [Powersurge/banned.video#7]
Not a really big deal in this case, just an idea.

We need to get this addon on some repos and on some people's radars too...hard since it's Alex, of course.

**Comment by Powersurge** [comment]
Yeah, that sounds like a great idea! Are you going to send me a PR? I also fixed the URL's for the m3u8 files.

**Comment by ilikenwf** [comment]
If I get around to it I will - nice job!

All I did was remove the "b" audio streams in a copied version of the actual stream links...somehow this fixed the audio sync.

**Comment by Powersurge** [comment]
Thank you! The "b" audio streams might have been something that my developer did a while back when this addon was having audio issues. I think the cause of the issue at that time was the way the audio was being encoded and it eventually got fixed.

**Comment by ilikenwf** [comment]
It's actually the way Infowars provides the streaims, they have two different audio streams and list them more than once in the m3u8, and for whatever reason this screws us up...it messes VLC up too.

**Comment by Powersurge** [comment]
Ok, that's what it was. I couldn't remember exactly.

**Comment by ilikenwf** [comment]
Yeah, and to be fair with more modification we could just load our own copy of the m3u up in the app from it's own directory but doing this way lets us adjust the files if we ever need to without having to update the whole addon.

**Comment by Powersurge** [comment]
I like that idea!

**Comment by ilikenwf** [comment]
Have you tried posting about this on https://old.reddit.com/r/Addons4Kodi/ or getting some repos to carry the addon? I realize reddit is super leftist but even so a lot of people watch Alex.

**Comment by Powersurge** [comment]
Lol, no unfortunately triggering leftists is just not my thing... but I am sure that there are plenty of other people out there that would jump at the opportunity.
